src/constants.js

```diff
@@ -57,3 +57,91 @@ export const PERCIPITATION_RAMP_COLORS = ['#A6BDDB', '#3690C0', '#034E7B']; // [
 export const MAPBOX_STYLE_DEFAULT = 'mapbox://styles/copernicus-forests/ck3eh5kj049be1cobcp1uh4vn';
 
 export const DEFAULT_LAYER_OPACITY = 60;
+
+export const FOOTER_MENU = [
+  {
+    title: 'Contact us',
+    link: '/contact-us',
+    fullUrl: 'https://climate.copernicus.eu/contact-us'
+  },
+  {
+    title: 'Cookies',
+    link: '/cookies',
+    fullUrl: 'https://climate.copernicus.eu/cookies'
+  },
+  {
+    title: 'Data Protection Officer',
+    link: '/data-protection-officer',
+    fullUrl: 'https://climate.copernicus.eu/data-protection-officer'
+  },
+  {
+    title: 'Data licence',
+    link:
+      'https://climate.copernicus.eu/sites/default/files/2018-04/20170117_Copernicus_License_V1.0.pdf',
+    fullUrl:
+      'https://climate.copernicus.eu/sites/default/files/2018-04/20170117_Copernicus_License_V1.0.pdf'
+  },
+  {
+    title: 'Disclaimer &amp; Privacy',
+    link: '/disclaimer-and-privacy',
+    fullUrl: 'https://climate.copernicus.eu/disclaimer-and-privacy'
+  }
+];
+
+export const FOOTER_SOCIAL_LINKS = [
+  {
+    class: 'twitter',
+    icon: '#icon-twitter',
+    link: 'https://twitter.com/CopernicusECMWF'
+  },
+  {
+    class: 'instagram',
+    icon: '#icon-instagram',
+    link: 'https://www.instagram.com/copernicusecmwf/'
+  },
+  {
+    class: 'slideshare',
+    icon: '#icon-slideshare',
+    link: 'https://www.slideshare.net/CopernicusECMWF'
+  }
+];
+
+export const HEADER_MENU_FIRST = [
+  {
+    title: 'News',
+    link: '/news',
+    fullUrl: 'https://climate.copernicus.eu/news'
+  },
+  {
+    title: 'Events',
+    link: '/events',
+    fullUrl: 'https://climate.copernicus.eu/events'
+  },
+  {
+    title: 'Press',
+    link: '/press-releases',
+    fullUrl: 'https://climate.copernicus.eu/press-releases'
+  },
+  {
+    title: 'Trends',
+    link: '/complete-list-tenders-issued-c3s',
+    fullUrl: 'https://climate.copernicus.eu/complete-list-tenders-issued-c3s'
+  },
+  {
+    title: 'Help &amp; Support',
```

Comment:
Please check the rendering of this HTML entity, it's rendering as "Help &amp; Support" in the browser.

Reply:
Fixed here: https://github.com/Vizzuality/copernicus-forest/pull/33/commits/d76cf8d043b4f4a776994fc7d880fe0791eefec5